### PR TITLE
Implement fee logic in events ticket dashboard table

### LINKED ISSUE #3007: Discount code logic: fees excluded from discount
Currently fees are being excluded from percentage discounts:
e.g even if the discount is 100%, the user still has to pay the fee.
selection_087

### PR DESCRIPTION
**Pull request by shubham-padia**
fixes #3007.
Includes fee logic in sales tab.
selection_095

### CODE REVIEW

File: app/views/users/ticket_sales.py [hunk 1]
```diff
@@ -81,23 +81,33 @@ def display_ticket_stats(event_id):
     for order in orders:
         if order.status == 'initialized':
             order.status = 'pending'
+        fees = DataGetter.get_fee_settings_by_currency(DataGetter.get_event(order.event_id).payment_currency)
         orders_summary[str(order.status)]['orders_count'] += 1
         orders_summary[str(order.status)]['total_sales'] += order.amount
+
```
Comment (SaptakS) [inline app/views/users/ticket_sales.py]: Why this extra line?

File: app/views/users/ticket_sales.py [hunk 2]
```diff
@@ -81,23 +81,33 @@ def display_ticket_stats(event_id):
...
         for order_ticket in order.tickets:
             discount = TicketingManager.get_discount_code(event_id, order.discount_code_id)
             orders_summary[str(order.status)]['tickets_count'] += order_ticket.quantity
             ticket = get_ticket(order_ticket.ticket_id)
             tickets_summary[str(ticket.id)][str(order.status)]['tickets_count'] += order_ticket.quantity
+            ticket_price = ticket.price
+            if fees and not ticket.absorb_fees:
+                order_fee = fees.service_fee * (ticket.price * order_ticket.quantity) / 100
```
Comment (SaptakS) [inline app/views/users/ticket_sales.py]: please make 100.0 just to ensure that it is always float value that is returned.

File: app/views/users/ticket_sales.py (hunk 1 above)
Comment (shubham-padia) [inline app/views/users/ticket_sales.py]: for better readability

File: app/views/users/ticket_sales.py (hunk 2 above)
Comment (shubham-padia) [inline app/views/users/ticket_sales.py]: :+1:

**Pull request merged**